src/Middleware/integrations/ordercloud.integrations.cardconnect/OrderCloudIntegrationsCardConnectService.cs

```diff
@@ -24,16 +25,22 @@ public class OrderCloudIntegrationsCardConnectConfig
         public string CadMerchantID { get; set; }
         public string EurMerchantID { get; set; }
     }
+    public enum AppEnvironment { Test, Staging, Production }
 
     public class OrderCloudIntegrationsCardConnectService : IOrderCloudIntegrationsCardConnectService
     {
         private readonly IFlurlClient _flurl;
+        private bool noAccountCredentials;
+        private AppEnvironment appEnvironment;
         public OrderCloudIntegrationsCardConnectConfig Config { get; }
 
-        public OrderCloudIntegrationsCardConnectService(OrderCloudIntegrationsCardConnectConfig config, IFlurlClientFactory flurlFactory)
+        public OrderCloudIntegrationsCardConnectService(OrderCloudIntegrationsCardConnectConfig config, string environment, IFlurlClientFactory flurlFactory)
         {
             Config = config;
-            _flurl = flurlFactory.Get($"https://{Config.Site}.{Config.BaseUrl}/");
+            // if no credentials are provided in Test and UAT, responses will be mocked.
+            noAccountCredentials = config == null || (config?.Authorization == null && config.AuthorizationCad == null) || (config?.Authorization == "" && config?.AuthorizationCad == "");
```

Comment:
A little shorter:

```csharp
noAccountCredentials = string.isNullOrEmpty(config?.Authorization) || string.isNullOrEmpty(config?.AuthorizationCad);
```

Reply:
fixed. had to do && since both Authorization and AuthorizationCad should be null for this I believe.